composer.json

```diff
@@ -20,7 +20,7 @@
 		"php": "^5.6 || ^7.0",
 		"kdyby/strict-objects": "^1.0",
 		"latte/latte": "^2.4.6@dev",
-		"nette/caching": "^2.5@dev",
+		"nette/caching": ">= 2.5.0 <= 2.5.6",
```

Comment:
I expect the BC break to be fixed in next version so this will be better.

`^2.5@dev !=2.5.7`